app/event-creator.tsx

```diff
@@ -1,14 +1,63 @@
 'use client'
 
-import React from "react";
+import React, { useState } from "react";
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
 
 export default function EventCreator() {
 
+   
+
     const [name, setName] = React.useState("");
     const [date, setDate] = React.useState("");
     const [calendarId, setCalendarId] = React.useState("");
 
+
+    function validateInput() {
+
+        var valid = true;
+
+        if (name.length == 0) {
+            const el = document.getElementById("name");
+            el.style.border = "2px solid red";
+
+            valid = false;
+            el.onchange = () => {el.style.border = "1px solid grey"};
+        }
+
+        if (date.length == 0) {
+            const el = document.getElementById("date");
+            el.style.border = "2px solid red";
+
+            valid = false;
+            el.onclick = () => { el.style.border = "1px solid grey" };
+        }
+
+        if (calendarId.length == 0) {
+            const el = document.getElementById("calendar");
+            el.style.border = "2px solid red";
+
+            valid = false;
+            el.onchange = () => { el.style.border = "1px solid grey" };
+        }
+
+        if (!valid) {
+
+            alert('Invalid Input')
+            return;
+        }
+
+
+        if (valid)
```

Comment:
Even though the brackets are unnecessary when there is a single line, maybe we should have them for consistency. What do you think?